examples/README.md

```diff
@@ -56,5 +64,15 @@ if (ret != DOTLOTTIE_SUCCESS) {
 }
 ```
 
+Finally, the player can be destroyed as follows:
+
+```C
+ret = dotlottie_destroy(player);
+if (ret != DOTLOTTIE_SUCCESS) {
+    fprintf(stderr, "Could not destroy dotlottie player\n");
+    return 1;
+}
```

Comment:
Do we need to add the cleanup in the c & skia examples ?

Reply:
It's not super important, as the application is not creating more than one player, but it would be good practice.